WireCalling/Sources/WireCallingUI/WireMeetings/MeetingsListViewModel.swift

```diff
@@ -0,0 +1,185 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+package import Foundation
+package import WireCallingDomain
+
+package final class MeetingsListViewModel: ObservableObject {
+    private typealias Strings = L10n.Localizable.WireMeetings.List
+
+    @Published var selectedTab: Tab = .next
+    @Published package var showAllNext: Bool = false
+
+    private let meetings: [Meeting]
```

Comment:
issue: I comment here but it applies to more things in this view model: there's a few things here that I think should go into a repository or similar (in WireCallingData target). For instance, something like:

```swift
protocol MeetingsRepository {

  func ongoingMeetings() -> [Meeting]

}
```

With this, the view model could further process returned meetings and expose them to the view for easy consumption.